runtime/test-runtime/src/lib.rs

```diff
@@ -100,7 +100,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_version: 1055,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 1,
+	transaction_version: 2,
```

Comment:
```suggestion
	transaction_version: 1,
```

This is only the test runtime. No bumping required.